src/irc/Ident.ts

```diff
@@ -146,15 +139,15 @@ class IdentSrv {
     private respond(sock: net.Socket, localPort: string, remotePort: string, username?: string) {
         let response;
         if (username) {
-            response = `${localPort},${remotePort}:USERID:UNIX:${username}\r\n`;
+            response = `${localPort}, ${remotePort}:USERID:UNIX:${username}\r\n`;
         }
         else {
-            response = `${localPort},${remotePort}:ERROR:NO-USER\r\n`;
-        }    
+            response = `${localPort}, ${remotePort}:ERROR:NO-USER\r\n`;
```

Comment:
You've added a space into the response strings here. Are you sure this is right?

Reply:
Nope, a mistake.